mobile/ios/device/ios-application-manager.ts

```diff
@@ -44,6 +44,96 @@ export class IOSApplicationManager extends ApplicationManagerBase implements Mob
 		}).future<void>()();
 	}
 
+	public getApplicationsLiveSyncSupportedStatus(): IFuture<void> {
+		return (() => {
+			let installationProxy = this.$injector.resolve(iOSProxyServices.InstallationProxyClient, { device: this.device });
+			let result = installationProxy.sendMessage({
+					"Command": "Browse",
+					"ClientOptions": {
+						"ApplicationType": "User",
+						"ReturnAttributes": [
+							"CFBundleIdentifier",
+							"IceniumLiveSyncEnabled"
+						]}
+				}).wait();
+
+			/*
+				Sample Result:
+				[{
+					"Total": 13,
+					"CurrentIndex": 0,
+					"CurrentAmount": 10,
+					"Status": "BrowsingApplications",
+					"CurrentList": [
+					{
+						"CFBundleIdentifier": "com.ebay.redlaserproper"
+					},
+					{
+						"CFBundleIdentifier": "com.apple.TestFlight"
+					},
+					{
+						"IceniumLiveSyncEnabled": true,
+						"CFBundleIdentifier": "com.telerik.TestSpecialChars"
+					},
+					{
+						"CFBundleIdentifier": "com.telerik.PlatformCompanion"
+					},
+					{
+						"IceniumLiveSyncEnabled": true,
+						"CFBundleIdentifier": "com.telerik.KendoUITabStrip1"
+					},
+					{
+						"IceniumLiveSyncEnabled": true,
+						"CFBundleIdentifier": "com.telerik.myAppNative1"
+					},
+					{
+						"CFBundleIdentifier": "com.ionic.viewapp"
+					},
+					{
+						"IceniumLiveSyncEnabled": true,
+						"CFBundleIdentifier": "com.telerik.samplepinchandzoom"
+					},
+					{
+						"CFBundleIdentifier": "com.telerik.7e4c83f6-4e40-420f-a395-ab3cd9f77afd.AppManager"
+					},
+					{
+						"IceniumLiveSyncEnabled": true,
+						"CFBundleIdentifier": "com.telerik.sampleinappbrowser"
+					}
+					]
+				},
+				{
+						"Total": 13,
+						"CurrentIndex": 10,
+						"CurrentAmount": 3,
+						"Status": "BrowsingApplications",
+						"CurrentList": [
+						{
+							"IceniumLiveSyncEnabled": true,
+							"CFBundleIdentifier": "com.telerik.KendoUIBlank9"
+						},
+						{
+							"IceniumLiveSyncEnabled": true,
+							"CFBundleIdentifier": "com.telerik.Blank1"
+						},
+						{
+							"IceniumLiveSyncEnabled": true,
+							"CFBundleIdentifier": "com.telerik.samplecapture"
+						}
+						]
+					}
+				]
+			 */
+
+			this.$logger.trace("Result when getting applications for which LiveSync is enabled: ", JSON.stringify(result, null, 2));
+			this.applicationsLiveSyncStatus = [];
+			_.each(result, (singleResult: any) => {
```

Comment:
you could optionally scramble up an interface for the result returned from the `installationProxy` with a mandatory `Status` attribute and all other ones - `Total`, `CurrentIndex`, `CurrentAmount`, `CurrentList` - optional.
Could help with understanding the communication as a whole.

Note that this is a minor suggestion and definitely not a merge-stopper